src/common/cast/remote-player-ui.ts

```diff
@@ -1,16 +1,30 @@
-// @flow
-import {Presets} from '@playkit-js/playkit-js-ui';
+import { Presets } from '@playkit-js/playkit-js-ui';
+
+// TODO - should come from UI
+class UIPreset {}
```

Comment:
can we use presets from UI now (after TS PR got merged)?

Reply:
fixed